tests/conftest.py

```diff
@@ -15,53 +16,58 @@
 
 # read in SQL for populating test data
 with open(os.path.join(os.path.dirname(__file__), "data.sql"), "rb") as f:
-    data_sql = f.read().decode("utf8")
+	data_sql = f.read().decode("utf8")
```

Comment:
Why does this one have double tab indentation?
I see that from this point, you have changed your indentation. It is not consistent with the rest of the codebase. Is it intentional?

Reply:
In my IDE it is a single tab indentation. I will fix this in the next update.